kolibri/core/assets/src/vue/progress-bar/index.vue

```diff
@@ -39,7 +36,7 @@
     },
     computed: {
       percent() {
-        return this.progress * 100;
+        return Math.min(this.progress * 100, 100);
```

Comment:
some of the percentages were being calculated incorrectly and returning values > 100. While that was wrong, I also didn't want it to blow up the progress bar component every time because it interfered with debugging.

This change displays inputted number (e.g. 350%) but caps the visual display at a full bar to prevent it from overflowing the boundaries.